.circleci/config.yml

```diff
@@ -158,6 +158,62 @@ jobs:
             docker run $DOCKER_IMG -version
       - *docker_push
 
+  publish-docker-stratum-bcm:
+    docker:
+      - image: stratumproject/build:build
+    environment:
+      - DOCKER_SCOPE: stratum/hal/bin/bcm/standalone/docker
+      - DOCKER_FILE: Dockerfile
+      - DOCKER_IMG: stratumproject/stratum-bcm
+    steps:
+      - run:
+          name: Install docker
+          command: |
+            curl -fsSL https://get.docker.com -o /tmp/get-docker.sh
+            sudo sh /tmp/get-docker.sh
+      - setup_remote_docker
+      - checkout
+      - *restore_bazel_cache
+      - *set_bazelrc
+      - run:
+          name: Build stratum_bcm
+          command: |
+            export CC='clang' CXX='clang++'
+            bazel build //stratum/hal/bin/bcm/standalone:stratum_bcm_package
+            tar xf bazel-bin/stratum/hal/bin/bcm/standalone/stratum_bcm_package.tar.gz -C $DOCKER_SCOPE
+      - *docker_login
+      - *docker_build
+      - *docker_push
+      - *save_bazel_cache
+
+  publish-docker-stratum-bcm-sdklt-3-16-56:
+    docker:
+      - image: stratumproject/build:build
+    environment:
+      - DOCKER_SCOPE: stratum/hal/bin/bcm/standalone/docker
+      - DOCKER_FILE: Dockerfile
+      - DOCKER_IMG: stratumproject/stratum-bcm:3.16.56
+    steps:
+      - run:
+          name: Install docker
+          command: |
+            curl -fsSL https://get.docker.com -o /tmp/get-docker.sh
+            sudo sh /tmp/get-docker.sh
```

Comment:
we can include `docker-ce-cli` to the build container I think

Reply:
That would still require the docker entry in apt.list? We could move it to build docker container, so when the other builds move to circleci remote docker, it's already there.